REFERENCE.md

```diff
@@ -1761,7 +1758,6 @@ Configure debian apt repo
 The following parameters are available in the `php::repo::debian` class:
 
 * [`location`](#location)
-* [`release`](#release)
```

Comment:
Looks odd for a patch release but it this seems to be a private class so I guess it is fine…